### PR TITLE
Welcome to tutorial section

### PR DESCRIPTION
**Pull request by bauerv**
First draft, comments welcome!

### CODE REVIEW

File: en/README.md [hunk 1]
```diff
@@ -4,6 +4,10 @@
 > This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License.
 > To view a copy of this license, visit https://creativecommons.org/licenses/by-sa/4.0/
 
+## Welcome
+Welcome to the Django Girls Tutorial! We are happy to see you here :) In this tutorial, we will take you on a journey under the hood of web technologies, offering you a glimps on all the bits and pieces that need to come together to make the web work as we know it.
```
Comment (helenst) [inline en/README.md]: "glimps on" -> "glimpse of"

File: en/README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@
 > This work is licensed under the Creative Commons Attribution-ShareAlike 4.0 International License.
 > To view a copy of this license, visit https://creativecommons.org/licenses/by-sa/4.0/
 
+## Welcome
+Welcome to the Django Girls Tutorial! We are happy to see you here :) In this tutorial, we will take you on a journey under the hood of web technologies, offering you a glimps on all the bits and pieces that need to come together to make the web work as we know it. 
+As all unknown things, this is going to be an adventure - but no worries, since you already worked up the courage to be here, you'll be just fine :)
```
Comment (helenst) [inline en/README.md]: "As all" -> "As with all"

**Comment by bauerv** [comment]
I've integrated the changes, thanks again :)

**Comment by helenst** [comment]
Thanks! :tulip:

**Pull request merged**